### PR TITLE
feat: add workflow for typo

### PR DESCRIPTION
**Pull request by xiaolou86**

### CODE REVIEW

File: .github/workflows/ci.yml
Comment (mkaput) [inline .github/workflows/ci.yml:1]: We do not want to employ such checks in our CI. Please revert changes in this file. I am fine with accepting just typo fixes that you did here.

**Review by mkaput — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml
Comment (xiaolou86) [reply .github/workflows/ci.yml:1]: ok, I will revert the changes.

**Comment by xiaolou86** [comment]
@mkaput Thanks for your review.

**Review by mkaput — APPROVED** [review]
Thx! :)

**Pull request merged**